infrastructure/kube/templates/bitcoin/electrumx/electrumx-statefulset.yaml

```diff
@@ -48,7 +49,7 @@ spec:
             - name: SSL_KEYFILE
               value: /mnt/electrum/cert/tls.key
             - name: SERVICES
-              value: tcp://:80,ssl://:443,ws://:8080,wss://:8443,rpc://0.0.0.0:8000
+              value: tcp://:50001,ssl://:50002,ws://:50003,wss://:50004,rpc://0.0.0.0:8000
```

Comment:
Do we need to update the readiness probe config? Is it something that requires yaml file update?

Reply:
No, we don't have to update them. We defined names for the ports:
```yaml
          ports:
            - name: tcp
              containerPort: 50001
```
So we can reference the ports by name:
```yaml
          readinessProbe:
            tcpSocket:
              port: tcp
```
Same we do in the [service (`targetPort`)](https://github.com/keep-network/keep-core/blob/11485daad086fa88fe9cc4b2c12c2aaa6a1b96d7/infrastructure/kube/templates/bitcoin/electrumx/electrumx-service.yaml#L13-L24):
```yaml
    - name: tcp
      port: 80
      targetPort: tcp
```

Reply:
Nice!